pkg/console/util.go

```diff
@@ -448,6 +448,15 @@ func doInstall(g *gocui.Gui, hvstConfig *config.HarvesterConfig, webhooks Render
 	if err := execute(ctx, g, env, "/usr/sbin/harv-install"); err != nil {
 		webhooks.Handle(EventInstallFailed)
 		printToPanel(g, fmt.Sprintf(installFailureMessage, defaultLogFilePath), installPanel)
+		if hvstConfig.Debug {
+			printToPanel(g, "support config is being generated as running in debug mode, this can take a few minutes...", installPanel)
+			fileSuffix := fmt.Sprintf("harvester_%s", rand.String(5))
+			scErr := executeSupportconfig(ctx, fileSuffix)
+			if scErr != nil {
+				printToPanel(g, fmt.Sprintf("support bundle collection failed %v", err), installPanel)
+			}
+			printToPanel(g, fmt.Sprintf("support config is available at /var/log/scc_%s.txz", fileSuffix), installPanel)
+		}
```

Comment:
LGTM.
nit: we can make the name more consistent. Either `Support config` or`Support bundle` for all strings.

Reply:
fixed it up